docs/en/build.md

```diff
@@ -92,9 +92,9 @@ Serial flasher config --->
 ```
 
 ### Partition Table
-It is not required to set a custom partition table. Per default the IDF will select `Single factory app, no OTA` and the firmware will create a partition for SPIFFS automatically which fills the remaining flash space.
+IDF's default partition table `Single factory app, no OTA` does not provide enough room for a firmware including large modules like e.g. `http` or `sodium`. To enable full feature sets, NodeMCU uses a custom partition table from `components/platform/partitions.csv`.
```

Comment:
I'm only just getting acquainted with ESP32. Hence, I don't yet understand the implications of this. You removed

> the firmware will create a partition for SPIFFS automatically which fills the remaining flash space.

because it's no longer true?

Reply:
As a novice I was simply missing the confirmation that yes, the rest is (still) being used for SPIFFS.